test/TestProjects/HeadAsBooleanTrue-LowLevel/Generated/HttpSuccessClient.cs

```diff
@@ -55,7 +55,7 @@ public HttpSuccessClient(AzureKeyCredential credential, Uri endpoint = null, Hea
         /// <summary> Return 200 status code if successful. </summary>
         /// <param name="options"> The request options. </param>
 #pragma warning disable AZC0002
-        public virtual async Task<Response<bool>> Head200Async(RequestOptions options = null)
+        public virtual async Task<Response<bool>> Head200Async(RequestOptions options)
```

Comment:
I believe `RequestOptions` should be optional in this case, since a response to a HEAD request doesn't have a response body.